### ISSUE TITLE
Unable to connect to panel via ESP32

### ISSUE THREAD

**Issue by ceedeebee907** [ParadoxAlarmInterface/pai#510]
## Alarm system
MG5050+ v1.25.002

### Git branch
REgular homeassistant release

## Question
Hoping someone could help me out here. I've inherited this system in a house I've recently purchased. It looks like the system was sometime in 2023. Anyway I've followed the instructions under the connection methods section with an LM2956. I've attached a snapshot of what the PAI interface in homassistant is seeing.

## Logs (if required)
`2024-10-14 21:37:32,467 - INFO     - PAI.paradox.paradox - Connecting to interface
2024-10-14 21:37:32,467 - INFO     - PAI.paradox.connections.ip.connection - Connecting. Try 1/3
2024-10-14 21:37:32,520 - INFO     - PAI.paradox.connections.connection - Connection established
2024-10-14 21:37:32,520 - INFO     - PAI.paradox.paradox - Connecting to Panel
2024-10-14 21:37:32,547 - INFO     - PAI.paradox.paradox - Panel Identified MG505+ª version 1.25 build 2
2024-10-14 21:37:32,547 - INFO     - PAI.paradox.paradox - Initiating panel connection
2024-10-14 21:37:37,557 - ERROR    - PAI.paradox.paradox - Timeout while connecting to panel. Is an other connection active?`

my pai add-on setup is below
`LOGGING_LEVEL_CONSOLE: 20
LOGGING_LEVEL_FILE: 40
CONNECTION_TYPE: IP
SERIAL_PORT: /dev/ttyUSB0
SERIAL_BAUD: 115200
IP_CONNECTION_HOST: 10.10.10.19
IP_CONNECTION_PORT: 6638
IP_CONNECTION_PASSWORD: paradox
KEEP_ALIVE_INTERVAL: 10
IO_TIMEOUT: 0.5
LIMITS:
  zone: auto
  user: 1-10
  door: ""
  pgm: 1-5
  partition: auto
  module: ""
  repeater: ""
  keypad: ""
  key-switch: ""
SYNC_TIME: true
SYNC_TIME_MIN_DRIFT: 120
PASSWORD: "0000"
MQTT_ENABLE: true
MQTT_HOST: core-mosquitto
MQTT_PORT: 1883
MQTT_KEEPALIVE: 60
MQTT_USERNAME: pai
MQTT_PASSWORD: pai
MQTT_HOMEASSISTANT_AUTODISCOVERY_ENABLE: true
COMMAND_ALIAS:
  arm: partition all arm
  disarm: partition all disarm
MQTT_COMMAND_ALIAS:
  armed_home: arm_stay
  armed_night: arm_sleep
  armed_away: arm
  disarmed: disarm
HOMEASSISTANT_NOTIFICATIONS_EVENT_FILTERS:
  - live,alarm,-restore
  - trouble,-clock
  - live,tamper
HOMEASSISTANT_PUBLISH_PARTITION_PROPERTIES:
  - target_state
  - current_state
HOMEASSISTANT_PUBLISH_ZONE_PROPERTIES:
  - open
  - tamper
PUSHBULLET_CONTACTS: []
PUSHBULLET_EVENT_FILTERS:
  - live,alarm,-restore
  - trouble,-clock
  - live,tamper
PUSHOVER_EVENT_FILTERS:
  - live,alarm,-restore
  - trouble,-clock
  - live,tamper
PUSHOVER_BROADCAST_KEYS: []
SIGNAL_CONTACTS: []
SIGNAL_EVENT_FILTERS:
  - live,alarm,-restore
  - trouble,-clock
  - live,tamper
GSM_CONTACTS: []
GSM_EVENT_FILTERS:
  - live,alarm,-restore
  - trouble,-clock
  - live,tamper
IP_INTERFACE_ENABLE: false
IP_INTERFACE_PASSWORD: paradox
DUMMY_EVENT_FILTERS: []
IP_CONNECTION_BARE: true
`

I've had a look through the forums and tried swapping the rx / tx pins however the panel doesn't even get identified by the add-on. Baud rate is set to 115200 as shown in the other forum posts. I've ordered a logic level converter as the 2nd serial port seems to be outputting a 5v signal. Any other ideas of what I could try?

**Comment by yozik04** [comment]
Panel identified this means that connection is established and baud rate is correct.
Problem can be in encrypted serial, but I am not sure.

**Comment by ceedeebee907** [comment]
Yeah fair. I'll reach out to the installer to see if the panel is locked.

**Comment by ceedeebee907** [comment]
bump? if anyone has any ideas, feel free to make suggestions

**Comment by yozik04** [comment]
Closing as a **duplicate of #349** — please follow that issue for updates and add any extra detail there.
